docs_outdoornav_user_manual/web_user_interface/ui_manual_mode.mdx

```diff
@@ -18,31 +18,54 @@ methods that can be used to stop the moving UGV.
 
 <center>
   <figure>
-    <img src="/img/outdoornav_images/ui_manual_mode_page.png" />
+    <img src="/img/outdoornav_images/ui_manual_mode_page_0_14_0.png" />
     <figcaption>Teleoperation Components</figcaption>
   </figure>
 </center>
 
 1.  **Error/Warning Bar:** A bar along the top of the page reporting
     the error or warning status of the Assisted Teleoperation feature.
-2.  **Joystick:** The joystick will allow the user to move the UGV
+2.  **User Connections:** Shows the number of users connected to Outdoor Nav.
+3.  **Joystick:** The joystick will allow the user to move the UGV
     manually from the UI. Motion can be sent to the UGV in 360°
     directions and the speed can be controlled by the distance of the
     joystick from its neutral position. Four fixed direction buttons are
     now present to move the UGV in purely forward/reverse directions or
     clockwise/counter-clockwise. Finally, we overlay an obstacle map
     on top of the joystick, representing obstacle distances from the
     UGV.
-3.  **Speedometer:** An indicator of the UGV's current linear speed.
-4.  **Sensitivity Scale:** A UGV-specific scale that controls the
+4.  **Speedometer:** An indicator of the UGV's current linear speed.
+5.  **Sensitivity Scale:** A UGV-specific scale that controls the
     maximum allowable UGV velocities at each level. The default maximum
     linear velocity is 1.0 meters per second and the maximum angular
     velocity is 0.5 radians per second. These can be changed in the
     [General Settings](./ui_overview.mdx#config-general-settings) page.
     The scale levels are 100%, 80%, 50% and 20%, with the default set
     to 80%.
-5.  **Teleop Assist Toggle:** The "Teleop Assist" toggle can be used to
+6.  **Teleop Assist Toggle:** The "Teleop Assist" toggle can be used to
     enable/disable the Assisted Teleoperation feature.
+7.  **UI Lockout:** The lockout toggle that will prevent the Joystick from 
+    sending commands to the UGV. The purpose of this is to help prevent 
```

Comment:
I would maybe say, The purpose is to prevent the unwanted users from teleoperating the UGV.